src/ExerciseRunner/CliRunner.php

```diff
@@ -145,7 +145,9 @@ public function run($fileName, OutputInterface $output)
 
         $output->writeTitle("\nOutput");
```

Comment:
fancy taking the `\n` out while your here and making the last write (L143) in the block above a writeLine instead ?